apps/friesian-server-helm/friesian-helm/readme.md

```diff
@@ -0,0 +1,98 @@
+| `recommender`                  | Recommender http server                                                                                                                                                                                                                                                                                                                                                  | Always On                              |
+| `recommender.serviceMonitor`   | Service Monitor for `recommender`. <br />**`prometheus` must be enabled (or installed)**                                                                                                                                                                                                                                                                                 | `recommender.serviceMonitor.enabled`   |
+| `recommender.ingress`          | [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) for `recommender`. <br />**An [ingress controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/) must be installed**                                                                                                                                         | `recommender.ingress.enabled`          |
+
+## Deploy the Friesian serving pipeline
+
+To install the chart with the release name `my-release` in namespace `friesian`:
+
+```bash
+helm upgrade --install --debug -n friesian my-release ./friesian-helm --create-namespace
+```
+
+After installation, follow Helm output to check whether Friesian serving works properly.
+
+## Cleanup
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+helm delete my-release -n friesian
+```
+
+The Helm chart
+uses [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)
+to monitor cluster.
+This stack creates a Service for scraping kubelet information (defined
+in `kube-prometheus-stack.prometheusOperator.kubeletService`) which isn't deleted when uninstalling.
+See [this issue](https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/1101) for a
+detailed explanation.
+
+To remove this service after uninstalling, run:
+
+```bash
+kubectl delete svc <release_name>-prometheus-kubelet -n kube-system
+
+# In this demo, replace <release_name> with my-release
+kubectl delete svc my-release-prometheus-kubelet -n kube-system
+```
+
+The commands remove all the Kubernetes components associated with the chart and delete the release.
```

Comment:
Make this ending sentence clearer.
The commands mean all the commands in this section or the commands only in the previous bash segment?